src/pages/_document.tsx

```diff
@@ -27,4 +30,31 @@ export default class MyDocument extends Document {
       sheet.seal();
     }
   }
+
+  // https://github.com/zeit/next.js/tree/master/examples/with-react-helmet
+  get helmetHtmlAttrComponents() {
+    return this.props.helmet.htmlAttributes.toComponent();
+  }
+
+  get helmetBodyAttrComponents() {
+    return this.props.helmet.bodyAttributes.toComponent();
+  }
+
+  get helmetHeadComponents() {
+    return Object.keys(this.props.helmet)
+      .filter(el => el !== 'htmlAttributes' && el !== 'bodyAttributes')
+      .map(el => this.props.helmet[el].toComponent());
+  }
+
+  render() {
+    return (
+      <html {...this.helmetHtmlAttrComponents}>
+        <Head>{this.helmetHeadComponents}</Head>
+        <body {...this.helmetBodyAttrComponents}>
+          <Main />
+          <NextScript />
+        </body>
+      </html>
+    );
+  }
```

Comment:
next.js has a function for this built in, in the past that is exclusively what I've used: https://nextjs.org/docs/api-reference/next/head

What was the rationale for using react-helmet? it seems like an unneeded package when the functionality is built into next.js, it adds [16kb/5.5kb](https://bundlephobia.com/result?p=react-helmet@6.0.0) to the final bundle.